src/common/crypto/mbedtls/drivers/psa_crypto.c

```diff
@@ -1,9176 +0,0 @@
-/*
```

Comment:
Why this files is removed? It should be updated instead.

Reply:
done